test/functional/apps/home/_sample_data.js

```diff
@@ -87,15 +106,40 @@ export default function ({ getService, getPageObjects }) {
         const tsvb = await find.existsByCssSelector('.vega-view-container');
         expect(tsvb).to.be(true);
       });
+
+      it('should launch sample logs data set dashboard', async ()=> {
+        await PageObjects.home.launchSampleDataSet('logs');
```

Comment:
I could not get this test to pass locally. At this point, the browser is on the `flights` dashboard so it can not find `logs` sample data card. I think you should just be able to change the `after` block (line 67) to `afterEach`

Reply:
@nreese I _think_ I should be good now, modified to `afterEach`

Reply:
ah, I know why it's failing